### PR TITLE
chore: allow manual trigger and minor recommended updates.

### PR DESCRIPTION
**Pull request by neverendingqs**

### CODE REVIEW

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -2,10 +2,12 @@ name: D2L License Checker CI
 on:
   - push
   - pull_request
+  - workflow_dispatch
```
Comment (neverendingqs) [inline .github/workflows/workflow.yml:5]: This allows us to manually trigger this workflow.

File: .github/workflows/workflow.yml [hunk 2]
```diff
@@ -2,10 +2,12 @@ name: D2L License Checker CI
...
 
 jobs:
   test:
     name: Test
+    runs-on: ubuntu-latest
```
Comment (neverendingqs) [inline .github/workflows/workflow.yml:10]: GitHub UI pointed out that this was missing, so I thought I'll fix it here.

File: .github/workflows/workflow.yml [hunk 3]
```diff
@@ -26,6 +28,7 @@ jobs:
 
   release:
     if: github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
```
Comment (neverendingqs) [inline .github/workflows/workflow.yml:31]: Similar: GitHub UI pointed out that this was missing, so I thought I'll fix it here.

**Comment by neverendingqs** [comment]
I also updated the branch protection rules to match the change from Travis CI to GitHub Actions.

**Review by BraydonKains — APPROVED** [review]
Thank you for the fixes. Didn't notice runs-on was required.

**Pull request merged**